v3api/transcripts_test.go

```diff
@@ -156,3 +156,41 @@ func TestTranscriptCancelError(t *testing.T) {
 	assert.NotNil(err)
 	assert.Equal("403: forbidden_error-You cannot cancel this transcript at this time.", err.Error())
 }
+
+func TestTranscriptEditingLink(t *testing.T) {
+	assert := assert.New(t)
+
+	defer gock.Off()
+	gock.New("https://api.3playmedia.com").
+		Get("/v3/transcripts/3633088/expiring_editing_link").
+		MatchParam("api_key", "api-key").
+		MatchParam("hours_until_expiration", "2").
+		Reply(200).
+		File("../fixtures/v3_transcript_link_200.json")
```

Comment:
It looks like this mock is not working in travis?

Reply:
Yeah, it was because I forgot to add the fixture file... 🤦‍♀️All good now.